.gitignore

```diff
@@ -0,0 +1 @@
+config.php
```

Comment:
Don't do this for Assignment 1, because me and the TA won't be able to run your project. Your assignment repo is a private repo so no big deal that the credentials are in the Git history. Plus, if anything were to happen to the database, it is easy to reset it.